sky/utils/cli_utils/status_utils.py

```diff
@@ -97,9 +97,19 @@ def show_status_table(cluster_records: List[Dict[str, Any]],
             autostop_minutes = spot.SPOT_CONTROLLER_IDLE_MINUTES_TO_AUTOSTOP
             click.echo(f'\n{colorama.Fore.CYAN}{colorama.Style.BRIGHT}'
                        f'{reserved_group_name}{colorama.Style.RESET_ALL}'
-                       f'{colorama.Style.DIM} (will be autostopped if idle for '
+                       f'{colorama.Style.DIM} (autostopped if idle for '
                        f'{autostop_minutes}min)'
                        f'{colorama.Style.RESET_ALL}')
+            reset = backend_utils.RESET_BOLD
+            click.echo(
+                f'{colorama.Style.DIM}  Directly interacting with the '
+                f'controller is typically not needed.{colorama.Style.RESET_ALL}'
+            )
+            click.echo(
+                f'  View spot jobs: {colorama.Style.BRIGHT}sky spot queue'
+                f'{reset}\n'
+                f'  View job logs:  {colorama.Style.BRIGHT}sky spot logs'
+                f'{reset}\n')
```

Comment:
This looks a bit verbose to me. How about:
```suggestion
            click.echo(
                f'{colorama.Style.DIM}  Directly interacting with the '
                f'controller is typically not needed.{colorama.Style.RESET_ALL} (Check  {colorama.Style.BRIGHT}sky spot --help{reset})'
            )
```

Reply:
<img width="1104" alt="Screen Shot 2022-11-30 at 12 28 45" src="https://user-images.githubusercontent.com/592670/204707752-ef38c311-a63b-44b8-b8dd-79b5b9fae379.png">

How about this? Reason: Putting them on one line seems to make the line too long and the command not noticeable.

Reply:
How about `For spot-related CLI, run: sky spot --help`?

Reply:
<img width="1102" alt="Screen Shot 2022-11-30 at 14 27 14" src="https://user-images.githubusercontent.com/592670/204723683-34f7260b-f99f-4ddc-8426-e1ea651d176c.png">

<img width="1098" alt="Screen Shot 2022-11-30 at 14 28 01" src="https://user-images.githubusercontent.com/592670/204723698-5a483397-90e0-4bfe-a0bf-34dddd6e2cf2.png">

Three versions. The bottom one looks best to me, as the command doesn't overlapped with `LAUNCHED`. Wdyt?

Reply:
I like the last one. Curious, why we don't add `--help`? That will explicitly telling the user it will show the helps (I think showing help without `--help` is non-standard behavior).

Or we can reword it by `Use spot jobs CLI that starts with: sky spot` (or include a linke to the docs?)

Reply:
Hmm, having the sentence below the `spot jobs CLI` hint does not read very smoothly. 
Can the sentence be shortened and combined with the `(autostopped if idle for 10 min)`, such as `(autostopped if idle for 10 min, direct interaction is typically not needed)`?

I think the overlapping of the `sky spot --help` with `LAUNCHED` should be fine?

Reply:
Reordered the command hint and the 'not needed' sentence.

I thought about putting it inside the parens after 'if idle for 10min', but that could be understood as "direct interaction to stop the cluster".